src/common/meta/src/error.rs

```diff
@@ -296,6 +296,37 @@ pub enum Error {
         error: serde_json::Error,
         location: Location,
     },
+
+    #[snafu(display("Invalid number of topics {}", num_topics))]
+    InvalidNumTopics {
+        num_topics: usize,
+        location: Location,
+    },
+
+    #[snafu(display(
+        "Failed to build an rskafka client, broker endpoints: {:?}",
```

Comment:
```suggestion
        "Failed to build a kafka client, broker endpoints: {:?}",
```

Reply:
I think we will only have one Kafka client, regardless of whether it is rskafka or something else.

Reply:
Updated.